package.json

```diff
@@ -1,18 +1,24 @@
 {
   "name": "electron-vue-next",
   "version": "0.0.0",
+  "main": "dist/source/index.js",
   "private": true,
   "description": "",
   "scripts": {
-    "dev:docs": "node scripts/dev.docs.js",
-    "dev": "node scripts/dev.js",
-    "build": "node scripts/build.js",
-    "build:docs": "vitepress build docs",
-    "build:dir": "cross-env BUILD_TARGET=dir node scripts/build.js",
-    "build:lite": "cross-env BUILD_TARGET=lite node scripts/build.js",
-    "build:production": "cross-env BUILD_TARGET=production node scripts/build.js",
+    "build": "(concurrently --kill-others-on-fail \"npm:build:main\" \"npm:build:renderer\") && npm run build:app",
+    "build:main": "cross-env NODE_ENV=production rollup --config rollup.config.js",
+    "build:renderer": "cross-env NODE_ENV=production vite build ./src/renderer --outDir ../../dist/source/renderer",
+    "build:app": "electron-builder build",
+
+
     "lint": "eslint --ext .ts,.vue,.js src scripts",
-    "lint:fix": "npm run lint -- --fix"
+    "lint:fix": "npm run lint -- --fix",
+
+
+    "dev": "concurrently -k \"npm:dev:main\" \"npm:dev:renderer\" \"npm:run:electron\"",
```

Comment:
Actually, in our legacy script. The async execution of vite server and rollup.watch is already concurrent. I think control them by scripting is more easy.

Reply:
The idea is not to do something in parallel, but to separate the processes from each other. So that you can easily add or remove something.

It is so. But in the old version it is somewhat difficult to add new processes. Imagine that I need to add a compilation of a web assembly. How do I do this? In the old version, I have to open the script, write a function to compile, and find the right place to run it.

In the new one, I can just add the npm command.